editor/editor_node.cpp

```diff
@@ -3155,24 +3155,36 @@ void EditorNode::_add_to_recent_scenes(const String &p_scene) {
 
 void EditorNode::_open_recent_scene(int p_idx) {
 
-	String base = "_" + ProjectSettings::get_singleton()->get_resource_path().replace("\\", "::").replace("/", "::");
-	Vector<String> rc = EDITOR_DEF(base + "/_recent_scenes", Array());
 
-	ERR_FAIL_INDEX(p_idx, rc.size());
+    String base = "_" + ProjectSettings::get_singleton()->get_resource_path().replace("\\", "::").replace("/", "::");
+
+    if(p_idx == recent_scenes->get_item_count() - 1){
```

Comment:
Style issue, there should be a space between `if` and `()`.

See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html